.github/workflows/ISSUE_TEMPLATE/other.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Issue Template
+about: Other issues template.
+title: ''
+labels: ''
```

Comment:
Maybe we could have a label other just to faster parse this types of issues?

Reply:
Not sure if that's really needed, at least for now we are not having difficulties managing new issues.

Let's evaluate that in the future

Reply:
Sure 👍